core/src/main/java/org/apache/gravitino/authorization/AuthorizationUtils.java

```diff
@@ -265,14 +268,26 @@ public static void checkSecurableObject(String metalake, MetadataObject object)
     }
   }
 
+  public static void checkDuplicatedNamePrivilege(Collection<Privilege> privileges) {
+    Set<Privilege.Name> privilegeNameSet = Sets.newHashSet();
+    for (Privilege privilege : privileges) {
+      if (privilegeNameSet.contains(privilege.name())) {
+        throw new IllegalPrivilegeException(
+            "Doesn't support duplicated privilege name %s with different condition",
```

Comment:
Can you output all the duplicated privilege names , not just one of them?

Reply:
This should be unnecessary. It will be allow and deny the privilege name.